core/creator/src/main/java/io/quarkus/creator/phase/nativeimage/NativeImagePhase.java

```diff
@@ -278,7 +278,7 @@ public void provideOutcome(AppCreator ctx) throws AppCreatorException {
             //TODO: use an 'official' image
             String image;
             if (dockerBuild.toLowerCase().equals("true")) {
-                image = "swd847/centos-graal-native-image-rc13";
+                image = "swd847/centos-graal-native-image-rc14";
```

Comment:
looks like these containers aren't available yet. We will have to wait for those before merging.

Reply:
👍 Sure thing.

Reply:
I think @cescoffier was going to setup a quay.io image repo so we have an official place to host these.

Reply:
Good, thanks for the update @stuartwdouglas. I'll be following the updates.

Reply:
> looks like these containers aren't available yet. We will have to wait for those before merging.

Done. Ready for another review.